### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Maria18000**
Added Media Queries in the mobile version CSS

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -424,3 +431,303 @@ footer hr {
   margin-top: 1vh;
   padding-top: 0%;
 }
+
+@media (min-width: 768px) {
```
Comment (leonardodiasb) [inline styles.css:431]: - [ ] Kindly review all of your margins, paddings, font-sizes, heights and widths, and every other attribute that you used `vh` or `vw` for sizing and change for the values that the Figma template provides (px in most cases). This is causing distortion on some sizes of screens and making a bad user experience when scrolling through your app.
image
image
image
image
image

File: styles.css [hunk 2]
```diff
@@ -424,3 +431,303 @@ footer hr {
...
+    justify-content: center;
+  }
+
+  .description {
+    width: 60%;
+  }
+
+  .card-body-head p {
+    margin-top: 0%;
+  }
```
Comment (leonardodiasb) [inline styles.css:517]: - [ ] Your regular projects should display a 'see project' button on hover. Kindly fix that.
image

File: styles.css [hunk 3]
```diff
@@ -227,7 +229,7 @@ form button {
 #see-project-small {
   width: 18.5vh;
   height: 7.3vh;
-  margin: 0 5vh;
+  margin-left: 0%;
   font-size: 2.58vh;
 }
```
Comment (leonardodiasb) [inline styles.css]: - [ ] Kindly correct the padding of your "See Project" on the desktop version.
image

File: styles.css [hunk 4]
```diff
@@ -424,3 +431,303 @@ footer hr {
...
+  .footer-top {
+    display: flex;
+    flex-direction: row;
+  }
+
+  form {
+    padding-top: 5%;
+    width: 70%;
+  }
+}
```
Comment (leonardodiasb) [inline styles.css]: - [ ] Your form margin/padding is not symmetrical. Kindly use the same dimensions the Figma template provides.
image
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Maria18000, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Maria18000** [comment]
Hi Leonardo,

The see project button was not included in the figma template. It was on
first card only, and I had to remove it for the other cards.
Let me know if it still needs to be added so I can proceed...

**Review by algerina — APPROVED** [review]
Hi @Maria18000 , 
Good job so far! :fire: 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed :heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR, however you can improve it:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSEs
 

 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**